src/figuresClasses.ts

```diff
@@ -1,19 +1,93 @@
-export interface Figure {
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle'
+}
+
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape,
+  color: Color,
+  getArea(): number
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape = Shape.Triangle ;
+
+  getArea(): number {
+    const semiperimeter: number = (this.a + this.b + this.c) / 2;
+
+    const area
+      = Math.sqrt(semiperimeter
+      * (semiperimeter - this.a)
+      * (semiperimeter - this.b)
+      * (semiperimeter - this.c));
 
+    return Math.floor(area * 100) / 100;
```

Comment:
you use rounding twice at your code, it would be better to create separate function